internal/xds/translator/testdata/out/xds-ir/jsonpatch.endpoints.yaml

```diff
@@ -7,6 +7,7 @@
             address: 1.2.3.4
             portValue: 50000
       loadBalancingWeight: 1
+    loadBalancingWeight: 50
```

Comment:
is this valid?

Reply:
Yes, I think it is https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/endpoint/v3/endpoint_components.proto#envoy-v3-api-msg-config-endpoint-v3-localitylbendpoints